Formula/azure-cli.rb

```diff
@@ -331,8 +336,8 @@ class AzureCli < Formula
   end
 
   resource "azure-mgmt-rdbms" do
-    url "https://files.pythonhosted.org/packages/33/c7/c50fa3c7fda5d58b313b247580a59dc78f6fb16ed50f843e355eb85bfe5a/azure-mgmt-rdbms-2.2.0.zip"
-    sha256 "f93344897a9bfa6ebc57dd0c10ad79602ff7965c322c65115e3f4b8584bbe1c7"
+    url "https://files.pythonhosted.org/packages/f7/bd/2aad57c3d6b93cc50b94b35bdfac21d45ebef10b472a88a6e8e8e9da26c8/azure-mgmt-rdbms-3.0.0rc1.zip"
+    sha256 "41dbb782a032253b477b230ca7299dbe8ae7b7c8315165735216f3e8eb2ecf29"
```

Comment:
Unless 3.0.0rc1 is needed to fix a security vulnerability we shouldn't have a release candidate resource. Can we stick with 2.2.0?

Reply:
@Juliehzl Can you confirm this? The version was bumped from here: https://github.com/Azure/azure-cli/pull/15177

Reply:
@rajsell could you help confirm if the new package fix security vulnerability?

Reply:
By the way, could package be beta version? If not, we will try to avoid such usage in the future.

Reply:
https://docs.brew.sh/Acceptable-Formulae#stable-versions we're a little more lenient with resources, but in general we like the software we ship to be stable (and I bet your customers do to)

Reply:
@Rylan12 does @Juliehzl's reply sounds reasonable to you? Can we merge this PR to unblock our customers first?

Reply:
I'll defer to Sean for this. @SMillerDev you had indicated to me that we only wanted release candidate resources that fix vulnerabilities. That doesn't seem the be the case here, but staying with the last stable release doesn't seem to be an option. Thoughts?

Reply:
I'm fine with using this, but please try and use stable versions in the future.